src/driver/Program.fs

```diff
@@ -668,13 +668,19 @@ let main argv =
                         else
                             return None
                     }
+                    let exitCode =                         
+                        if result.ExitCode = 0 && result.StandardError.Length <> 0 then
+                            -1
```

Comment:
> -1 [](start = 27, length = 3)

Please define a constant that's not used in the engine (e.g. -2) and use it here, this will help flag such cases in telemetry.